pkg/cover/report_test.go

```diff
@@ -73,6 +80,15 @@ func TestReportGenerator(t *testing.T) {
 			CFlags:    []string{"-fsanitize-coverage=trace-pc"},
 			DebugInfo: true,
 		},
+		{
+			Name:               "mismatch-pcs",
+			AddCover:           true,
+			AddBadPc:           true,
+			CFlags:             []string{"-fsanitize-coverage=trace-pc"},
+			DebugInfo:          true,
+			Result:             `.* do not have matching coverage callbacks`,
+			GCCOverridesResult: true,
```

Comment:
I think something like `SkipGccTailCallBug: true` that would just disable this test for all `gcc < 14` would be a bit simpler.

Reply:
Done. I also changed the wording a little, including flag names.